README.md

```diff
@@ -1,11 +1,23 @@
-# Open 3D Engine Networking Special Interest Group
+# Open 3D Engine Special Interest Group on Networking
 
-Materials for the Open 3D Engine Foundation Networking Special Interest Group (O3DE Network SIG).
-We manage the networking components including foundational pieces, transport, Multiplayer, client/server interactions and Cloud connectivity pieces of O3DE.
+This repoistory holds materials for the Open 3D Engine Foundation Special Interest Group for Networking (SIG-Network).
+
+We manage the networking components for O3DE including foundational networking layers, transport, Multiplayer, client/server interactions and Cloud connectivity pieces of O3DE. See the [SIG's charter](governance/SIG%20Network%20Charter.md) for further details.
```

Comment:
```suggestion
We manage the networking components for O3DE including foundational networking layers, transport, Multiplayer, client/server interactions and Cloud connectivity. See the [SIG's charter](governance/SIG%20Network%20Charter.md) for further details.
```

Recommend removing pieces of O3DE at the end given networking components for O3DE is specified up front. Alternatively, a clearer separation between networking components and cloud connected components. Depends on the intent here.